dotCMS/src/main/java/com/dotmarketing/business/VersionableFactoryImpl.java

```diff
@@ -385,6 +385,23 @@ public Optional<ContentletVersionInfo> findAnyContentletVersionInfo(final String
 		return !result.isEmpty() ? Optional.of(result.get(0)) : Optional.empty();
 	}
 
+	@Override
+	public Optional<ContentletVersionInfo> findAnyContentletVersionInfo(final String identifier, final boolean deleted)
```

Comment:
[24](https://dotcms.slack.com/archives/C028Z3R2D/p1667409850426659)
in favor of avoiding flags we can translate this
public Optional<ContentletVersionInfo> findAnyContentletVersionInfo(final String identifier, final boolean deleted)
to
findAnyContentletVersionInfoIncludeDeleted(id) {
    return indAnyContentletVersionInfo(id, true);
}
findAnyContentletVersionInfoExcludeDeleted(id) {
    return indAnyContentletVersionInfo(id, false);
}
And make the one that receives 2 params private?